src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -140,6 +138,11 @@ public void addMusicianToBand(int bandIndex, int musicianIndex) {
         addressBook.addMusicianToBand(bandIndex, filteredMusicians.get(musicianIndex));
     }
 
+    @Override
+    public void removeMusicianFromBand(int bandIndex, int musicianIndex) {
+        addressBook.removeMusicianFromBand(bandIndex, filteredMusicians.get(musicianIndex));
```

Comment:
I see that `removeMusicianFromBand` successfully removes the musician from the address book (storage). However, I realised that the UI does not immediately reflect the removal of musician from the band. Hence, may I suggest after calling this method, we call the `updateFilteredBandMusicianList()` of `ModelManager` so that hopefully the musicians get removed from the musician panel. 

I only implemented the above said method in a recently merged PR so you could find it while merging the conflicts!